website/versioned_docs/version-stable/webstorm.md

```diff
@@ -18,9 +18,9 @@ For older IDE versions, please follow the instructions below.
 
 To automatically format your files using `prettier` on save, you can use a [File Watcher](https://plugins.jetbrains.com/plugin/7177-file-watchers).
 
-Go to _Preferences | Tools | File Watchers_ and click **+** to add a new watcher.
+In the _Settings/Preferences_ dialog (`⌘,` on Mac or `Ctrl+Alt+S` on Windows), click **File Watchers** under **Tools** and click **+** to add a new watcher.
 
-In Webstorm 2018.2, select Prettier from the list, review the configuration, add any additional arguments if needed, and click OK.
+In Webstorm 2018.3, select Prettier from the list, review the configuration, add any additional arguments if needed, and click OK.
```

Comment:
I'd suggest "In WebStorm 2018.2 or above" or remove that bit completely and start with "Select Prettier".